conda_smithy/tests/test_lint_recipe.py

```diff
@@ -261,6 +264,24 @@ def test_cli_environ(self):
             out, _ = child.communicate()
             self.assertEqual(child.returncode, 0, out)
 
+    def test_unicode(self):
+        """
+        Tests that unicode does not confuse the linter.
+        """
+        with tmp_directory() as recipe_dir:
+            with io.open(os.path.join(recipe_dir, 'meta.yaml'), 'wt') as fh:
+                fh.write(u"""
+                    package:
+                        name: 'test_package'
+                    build:
+                        number: 0
+                    about:
+                        home: something
+                        license: something else
+                        summary: αβɣ
```

Comment:
Sure. It really does not matter for this particular issue but maybe for some future issue.